src/OpenTelemetry/Sdk.cs

```diff
@@ -47,6 +48,11 @@ static Sdk()
         /// </summary>
         public static bool SuppressInstrumentation => SuppressInstrumentationScope.IsSuppressed;
 
+        /// <summary>
+        /// Gets meter to create instruments for reporting metrics on Sdk internal state.
+        /// </summary>
+        internal static Meter SdkInternalStateMeter { get; } = new Meter("OpenTelemetry.Sdk");
```

Comment:
nit: Could probably just call it something like "InternalMeter" especially because you have to use the class name to reference it. For example `Sdk.InternalMeter.Create...` reads nicely.